### PR TITLE
Added German translations.

### PR DESCRIPTION
**Pull request by juliquiron**
The strings are coming from the integration branch

https://app.asana.com/0/633061025239778/759499408576339

### CODE REVIEW

File: translations/ssofact.de.po [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+"Last-Translator: NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <EMAIL@ADDRESS>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: modules/contrib/ssofact/ssofact.module:23
+msgid "About"
+msgstr "Ãœber das Modul"
```
Comment (sun) [inline translations/ssofact.de.po]: The file does not seem to be encoded in UTF-8/Unicode… How was it created?

File: translations/ssofact.de.po [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+msgid "Client secret"
+msgstr "Client Secret"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:50
+msgid "Server domain"
+msgstr "Server-Domain"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:58
+msgid "Scope"
+msgstr "Umfang"
```
Comment (sun) [inline translations/ssofact.de.po]: This is a weird translation, but these technical OpenID Connect terms are hard to translate…

Can we remove all the translations that are not necessary right now and only add translations for the relevant strings in the user login form?

File: translations/ssofact.de.po [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:114
+msgid "Email address"
+msgstr "E-Mailadresse"
```
Comment (sun) [inline translations/ssofact.de.po]: Translated by Drupal core already.

File: translations/ssofact.de.po [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:129
+msgid "Password"
+msgstr "Passwort"
```
Comment (sun) [inline translations/ssofact.de.po]: Translated by Drupal core already.

File: translations/ssofact.de.po [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:137
+msgid "Stay logged-in"
+msgstr "Eingeloggt bleiben"
```
Comment (sun) [inline translations/ssofact.de.po]: Needs to be the same string as in https://stage-login.stimme.de/ — "Angemeldet bleiben"

File: translations/ssofact.de.po [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+msgid "Forgot password?"
+msgstr "Passwort vergessen?"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:151 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:122 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:122
+msgid "Send password reset instructions via email."
+msgstr "Anleitung zur RÃ¼cksetzung des Passworts per Mail senden."
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:165 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:108 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:108
+msgid "Log in"
+msgstr "Einloggen"
```
Comment (sun) [inline translations/ssofact.de.po]: Needs to be the same string as in https://stage-login.stimme.de/ — "Anmelden"

File: translations/ssofact.de.po [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:123 modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:101
+msgid "Your email address"
+msgstr "Ihre E-Mailadresse."
```
Comment (sun) [inline translations/ssofact.de.po]: Seemingly translated by Drupal core already now?

If it is kept, then it needs to be "E-Mail-Adresse" and without trailing period.

File: translations/ssofact.de.po [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:144
+msgid "I accept terms and conditions"
+msgstr "Ich akzeptiere die GeschÃ¤ftsbedingungen"
```
Comment (sun) [inline translations/ssofact.de.po]: This is a longer text (see https://app.asana.com/0/633061025239778/732469091269190) and I doubt it makes sense to reverse-translate it into English

File: translations/ssofact.de.po [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:161 modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:119
+msgid "Sign up"
+msgstr "Anmelden"
```
Comment (sun) [inline translations/ssofact.de.po]: "Registrieren"

File: translations/ssofact.de.po [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+msgid "User register (ssoFACT)"
+msgstr "Nutzerregistrierung (ssoFACT)"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:14
+msgid "User server register (ssoFACT)"
+msgstr "Nutzerregistrierung Server (ssoFACT)"
+
+#: modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:63
+msgid "RFBE key"
+msgstr "RFBE SchlÃ¼ssel"
```
Comment (sun) [inline translations/ssofact.de.po]: All strings starting from here are internal/administrative and should be removed

**Review by sun — CHANGES_REQUESTED** [review]

**Comment by juliquiron** [comment]
All fixes applied, ready for new review. 

Changed the base to `paywall-juli` because in master there was not all the strings. I will try to move forward the merging of the paywall branch.

**Comment by sun** [comment]
The branch paywall-juli is not in a functional state (fatal error), so that isn't a good/suitable base for this PR.  Perhaps `paywall-states-barriers` would be better?

File: src/Form/SsofactRegisterForm.php [hunk 11]
```diff
@@ -129,7 +129,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['privacy'] = [
       '#type' => 'checkbox',
       '#required' => TRUE,
-      '#title' => $this->t('I accept terms and conditions'),
+      '#title' => $this->t('Please note the general <a href=":url_data">data protection regulations</a> and the <a href=":url_info">information obligation</a>.', [
+        ':url_data' => '/node/2',
+        ':url_info' => 'https://www.stimme-medien.de/metanavigation/informationspflichten/',
```
Comment (sun) [inline src/Form/SsofactRegisterForm.php:134]: Hardcoding these is pretty bad… but I do not see a quick solution for this right now, so we need to come back to this later.

**Review by sun — APPROVED** [review]
Removed the translations for administrative strings.

**Pull request merged**